src/backend/api/Fusion.Resources.Domain/Services/ProfileServices.cs

```diff
@@ -51,24 +50,38 @@ public async Task<DbExternalPersonnelPerson> RefreshExternalPersonnelAsync(Perso
             if (resolvedPerson == null)
                 throw new PersonNotFoundError(personId.OriginalIdentifier);
 
+            bool hasChanged = false;
+
             if (profile != null)
             {
-                resolvedPerson.AccountStatus = profile.GetDbAccountStatus();
-                resolvedPerson.AzureUniqueId = profile.AzureUniqueId;
-                resolvedPerson.JobTitle = profile.JobTitle;
-                resolvedPerson.Name = profile.Name;
-                resolvedPerson.Phone = profile.MobilePhone ?? string.Empty;
-                resolvedPerson.PreferredContractMail = profile.PreferredContactMail;
-                resolvedPerson.IsDeleted = false;
+                if (resolvedPerson.AccountStatus != profile.GetDbAccountStatus() || resolvedPerson.AzureUniqueId != profile.AzureUniqueId || resolvedPerson.IsDeleted)
```

Comment:
Why only save changes when the account status has changed? 
This would not include if jobtitle or just phone has changed?

I think you could try to do?
```csharp
                var changes = db.Entry(dbUser)?.Properties
                    .Where(x => x.IsModified)
                    .ToList();
```